src/Classes/Particle/Updater.ts

```diff
@@ -162,25 +207,33 @@ export class Updater {
                 container.particles.array.splice(idx, 1);
 
                 /* remove the canvas if the array is empty */
-                const clickMode = options.interactivity.events.onclick.mode;
+                const clickMode = options.interactivity.events.onClick.mode;
 
                 if (!container.particles.array.length && !Utils.isInArray(ClickMode.push, clickMode)) {
                     container.destroy();
                 }
             }
         } else {
-            if (particle.position.x - particle.radius > container.canvas.dimension.width - particle.offset.x) {
+            const nextPos = {
+                x_left: particle.position.x - particle.radius,
+                x_right: particle.position.x + particle.radius,
+                y_top: particle.position.y - particle.radius,
+                y_bottom: particle.position.y + particle.radius,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'y_bottom' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5139404)